packages/ui/src/helpers/authenticator/__tests__/utils.test.ts

```diff
@@ -74,3 +79,16 @@ describe('configureComponent', () => {
     );
   });
 });
+
+describe('isValidEmail', () => {
```

Comment:
Since `isValidEmail` accepts `undefined` as its parameter can we add a test case for it?